scene/gui/rich_text_label.cpp

```diff
@@ -2216,6 +2220,35 @@ RichTextLabel::Item *RichTextLabel::_find_indentable(Item *p_item) {
 	return indentable;
 }
 
+List<RichTextLabel::Item *> RichTextLabel::_get_table_frames(ItemTable *p_table) const {
+	List<RichTextLabel::Item *> search_list;
+	List<RichTextLabel::Item *> frames;
+
+	// Add all children to search queue
+	for (const List<Item *>::Element *E = p_table->subitems.front(); E; E = E->next()) {
+		search_list.push_front(E->get());
+	}
+
+	while (!search_list.is_empty()) {
+		Item *c = search_list.back()->get();
+		search_list.pop_back();
+
+		ERR_CONTINUE((c->type != ITEM_CONTEXT) && (c->type != ITEM_FRAME));
+
+		if (c->type == ITEM_FRAME)
+			frames.push_front(static_cast<Item *>(c));
+
+		// If item is a context item, search through its children
+		else if (c->type == ITEM_CONTEXT) {
```

Comment:
```suggestion
		if (c->type == ITEM_FRAME) {
			frames.push_front(static_cast<Item *>(c));
		} else if (c->type == ITEM_CONTEXT) {
			// If item is a context item, search through its children.
```
Always use brackets, and also group if/else